.github/workflows/darwin.yaml

```diff
@@ -98,7 +98,9 @@ jobs:
             - name: Build OTA image files with software version number 5
               run: |
                   scripts/examples/gn_build_example.sh examples/ota-requestor-app/linux out/debug chip_config_network_layer_ble=false non_spec_compliant_ota_action_delay_floor=0 chip_device_config_device_software_version=5 chip_device_config_device_software_version_string='"5.0"'
-                  cp out/debug/chip-ota-requestor-app /tmp/ota-test005-raw-image
+                  cp out/debug/chip-ota-requestor-app /tmp/ota-test006-raw-image
```

Comment:
Maybe we should just call this /tmp/ota-raw-image-v5 and call the other one /tmp/ota-raw-image-v10 to make it clearer what they actually are, especially since we use them both in test007 and test008, and use one of them in test005?

Followup is fine for this part; your choice.

Reply:
yeah i thought about that and I agree. its better.

Reply:
done.